### DISCUSSION TITLE
ros2 module?

### DISCUSSION THREAD

**Discussion by vikgus1** [facontidavide/PlotJuggler#885] [Q&A]
Hi,

I wanted to fork the foxy branch and add some stuff related to topic remapping. When checking the source code of the foxy branch for the ros2 module, I couldn't find any includes of rclcpp. How are the ros2 publishers and subscribers managed? Is there a subrepo or similar used for them?

![image](https://github.com/facontidavide/PlotJuggler/assets/102746544/bd6bec7e-b012-43a6-a89a-42db5d600811)

Best regards,
Viktor

**Comment by lucasw** [comment]
Take a look at [plotjuggler-ros-plugins](https://github.com/PlotJuggler/plotjuggler-ros-plugins), [DataStreamROS2](https://github.com/PlotJuggler/plotjuggler-ros-plugins/tree/rolling/plugins/DataStreamROS2) in particular
